model/email_notification_info.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+type EmailNotificationInfoInterface interface {
```

Comment:
What would you think of renaming the interface to `EmailNotification` then the method could just be named `ToEmail`? 

That way it largely resembles our `model.Event` interface as well.